shared/src/business/useCases/docketEntry/fileCourtIssuedDocketEntryInteractor.js

```diff
@@ -158,20 +167,34 @@ exports.fileCourtIssuedDocketEntryInteractor = async ({
     });
 
   const saveItems = [
-    applicationContext.getPersistenceGateway().createUserInboxRecord({
-      applicationContext,
-      workItem: workItem.validate().toRawObject(),
-    }),
-    applicationContext.getPersistenceGateway().createSectionInboxRecord({
-      applicationContext,
-      workItem: workItem.validate().toRawObject(),
-    }),
     applicationContext.getPersistenceGateway().updateCase({
       applicationContext,
       caseToUpdate: caseEntity.validate().toRawObject(),
     }),
   ];
 
+  if (isUnservable) {
```

Comment:
Should we add a new test for this? I didn't see one added to the test file.

Reply:
Based on the coverage report, it looks like everything is covered apart from the following conditional date `filingDate: documentEntity.filingDate || createISODateString(),` which is no longer needed. I'll remove the default value.